packages/agents-openai/src/openaiResponsesModel.ts

```diff
@@ -525,7 +526,10 @@ function getInputItems(
           type: 'code_interpreter_call',
           id: item.id!,
           code: item.providerData?.code ?? '',
-          results: item.providerData?.results ?? [],
+          // This property used to be results, so keeping both for backward compatibility
+          // That said, this property cannot be passed from a user, so it's just API's internal data.
+          outputs:
```

Comment:
this should be safe; the underlying sdk made this breaking changes but agents sdk does not touch this property so far